client/src/account/accountStyles.css

```diff
@@ -55,3 +65,26 @@ a {
 .cancelButton {
    margin-top: auto;
 }
+
+/* these styles change the size of the text in the rentals */
+@media (max-width:550px) {
+  h4 {font-size:.60rem; }
+  h3 {font-size:.75rem; }
+  h2 {font-size:.9rem; }
+  h1 {font-size:1rem; }
+}
+
+@media (min-width: 551px) {
+  h4 {font-size:.9rem; }
+  h3 {font-size:1rem; }
+  h2 {font-size:1rem; }
+  h1 {font-size:1.25rem; }
+}
+
+@media (min-width: 950px) {
+  h4 {font-size:1rem;}
+  h3 {font-size:1.2rem;}
+  h2 {font-size:1.2rem;}
+  h1 {font-size:1.3rem;}
+
+}
```

Comment:
if you have time, I'd suggest just reviewing the spacing before and after many curly braces in this file to make it a bit more consistent.